packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -423,7 +423,7 @@ export function TableNG(props: TableNGProps) {
     return (
       <>
         <MenuItem
-          label="Inspect value"
+          label={t('grafana-ui.table.inspect-menu-label', 'Inspect value')}
```

Comment:
This has the same value as `grafana-ui.table.cell-inspect-tooltip`, could it be a good idea to unify them?

Reply:
hmm, i don't think so 🤔 unless i'm mistaken these are 2 separate labels for separate actions, which coincidentally have the same value. think we've always said we should keep these cases separate - that way we can independently change one or the other easily.

Reply:
Ah, since the component name was `TableCellNG` and this is `TableNG` I thought the actions were related, but if not, ignore this 😉

Reply:
they're related, but they don't quite do the same thing. one opens the inspect drawer, the other opens the inspect drawer but filters to a specific cell 🤔 it could go either way tbh, but it feels safer to keep them separate then we have full flexibility 🤷